manifests/role/webhost/fulcrum_www_and_app.pp

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2019-2020 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# Webhost for fulcrum.org (extracted from www.lib, so some extraneous stuff)
+#
+# @example
+#   include nebula::role::webhost::fulcrum_www_and_app
+class nebula::role::webhost::fulcrum_www_and_app (
+  String $private_address_template = '192.168.0.%s',
+  Hash $hosts = {}
+) {
+  include nebula::role::umich
+  include nebula::role::fulcrum::app_host
+  include nebula::profile::www_lib::register_for_load_balancing
+  include nebula::profile::networking::firewall::http
+
+  create_resources('host',$hosts)
+
+  include nebula::profile::www_lib::apache_minimum
+  include nebula::profile::www_lib::fulcrum_apache
```

Comment:
Ok. So a regular www-lib vm just pulls in `apache`, which in turn pulls in both of these and also adds its own stuff. A fulcrum www/app vm pulls in both of these but without everything added by `apache`. I propose:

```
fulcrum_www_and_app.pp:
  include nebula::profile::www_lib::apache::base
  include nebula::profile::www_lib::apache::fulcrum

www_lib_vm.pp:
  include nebula::profile::www_lib::apache::base
  include nebula::profile::www_lib::apache::fulcrum
  include nebula::profile::www_lib::apache::misc
```

Reply:
I like these names and the suggestion to flatten includes. This is now complete in the latest commit. It also clarifies what default_access is and how it is used (whether we want to continue its use that way).